client/core/locale_ntfn.go

```diff
@@ -346,12 +346,12 @@ var ptBR = map[Topic]*translation{
 		template: "Conectado com carteira para completar o registro em %s, mas falha ao destrancar: %v",
 	},
 	// [ticker, error]
-	TopicWithdrawError: {
+	TopicSendError: {
 		subject:  "Erro Retirada",
 		template: "Erro encontrado durante retirada de %s: %v",
```

Comment:
We don't have much of a system for this, but maybe comment with `// RETRANSLATE`.  If the message were completely changed, we'd just go back to English, but I actually think we're better off keeping the previous translations even if they still say "withdraw" because it still gets the point across and isn't technically incorrect.